.editorconfig

```
@@ -0,0 +1,8 @@
+root = true
+
+[*.{rb}]
+indent_style = space
+indent_size = 4
```

~We should paint the bike shed chartreuse~ Ruby style is almost universally 2-space indentation.

```suggestion
indent_size = 2
```